### PR TITLE
Version and CHANGELOG updates for CMake Tools #1.4 release.

### PR DESCRIPTION
**Pull request by andreeis**
Version and CHANGELOG updates for CMake Tools #1.4 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
 # What's New?
 
+## 1.4
+Improvements:
+- Commit text. [PR #pullNumber](https://github.com/microsoft/vscode-cmake-tools/pull/pullNumber) [@gitHubUser](https://github.com/gitHubUser)
```
Comment (bobbrow) [inline CHANGELOG.md:5]: The copy/paste template can be removed. 😃

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+- Use a shell script to set environment variables for a kit. [#809](https://github.com/microsoft/vscode-cmake-tools/issues/809) [@pisker](https://github.com/pisker)
+- Improvements of the status bar UI. [PR #1200](https://github.com/microsoft/vscode-cmake-tools/pull/1200) [@SchweizS](https://github.com/SchweizS)
+- Add context menu for CMakeLists. [#741](https://github.com/microsoft/vscode-cmake-tools/issues/741) [@SchweizS](https://github.com/SchweizS)
+- Support partial CMake Tools activation for non cmake repos. [PR #1206](https://github.com/microsoft/vscode-cmake-tools/pull/1206)
+- Support ARM IntelliSense modes. [#1155](https://github.com/microsoft/vscode-cmake-tools/issues/1155)
+- Support GNU language standards. [#1208](https://github.com/microsoft/vscode-cmake-tools/issues/1208)
+- Add indication of active workspace to project outline. [#1183](https://github.com/microsoft/vscode-cmake-tools/issues/1183) [@SchweizS](https://github.com/SchweizS)
+
+Bug Fixes:
+- Commit text. [#issueNumber](https://github.com/microsoft/vscode-cmake-tools/issues/issueNumber) [@gitHubUser](https://github.com/gitHubUser)
```
Comment (bobbrow) [inline CHANGELOG.md:17]: The copy/paste template can be removed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
 
+## 1.4
+Improvements:
+- Commit text. [PR #pullNumber](https://github.com/microsoft/vscode-cmake-tools/pull/pullNumber) [@gitHubUser](https://github.com/gitHubUser)
+- Documentation updates. [PR #1130](https://github.com/microsoft/vscode-cmake-tools/pull/1130) [@zalava](https://github.com/zalava)
+- Add support for per-folder browse path. [#1073](https://github.com/microsoft/vscode-cmake-tools/issues/1073)
+- Use a shell script to set environment variables for a kit. [#809](https://github.com/microsoft/vscode-cmake-tools/issues/809) [@pisker](https://github.com/pisker)
+- Improvements of the status bar UI. [PR #1200](https://github.com/microsoft/vscode-cmake-tools/pull/1200) [@SchweizS](https://github.com/SchweizS)
+- Add context menu for CMakeLists. [#741](https://github.com/microsoft/vscode-cmake-tools/issues/741) [@SchweizS](https://github.com/SchweizS)
+- Support partial CMake Tools activation for non cmake repos. [PR #1206](https://github.com/microsoft/vscode-cmake-tools/pull/1206)
```
Comment (bobbrow) [inline CHANGELOG.md:11]: This is issue #1167

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+- Support ARM IntelliSense modes. [#1155](https://github.com/microsoft/vscode-cmake-tools/issues/1155)
+- Support GNU language standards. [#1208](https://github.com/microsoft/vscode-cmake-tools/issues/1208)
+- Add indication of active workspace to project outline. [#1183](https://github.com/microsoft/vscode-cmake-tools/issues/1183) [@SchweizS](https://github.com/SchweizS)
+
+Bug Fixes:
+- Commit text. [#issueNumber](https://github.com/microsoft/vscode-cmake-tools/issues/issueNumber) [@gitHubUser](https://github.com/gitHubUser)
+- Skip over debugger guessing logic if cmake.debugConfig explicitly sets miDebuggerPath. [#1060](https://github.com/microsoft/vscode-cmake-tools/issues/1060)
+- Normalize all paths sent to CppTools. [#1099](https://github.com/microsoft/vscode-cmake-tools/issues/1099)
+- Add support for Objective-C and Objective-C++. [#1108](https://github.com/microsoft/vscode-cmake-tools/issues/1108) [@marksisson](https://github.com/marksisson)
+- Update extension id. [PR #1116](https://github.com/microsoft/vscode-cmake-tools/pull/1116) [@ChristianS99](https://github.com/ChristianS99)
```
Comment (bobbrow) [inline CHANGELOG.md:21]: This is issue #1045.

Can you update the text to say "Update the configuration provider id" instead?  This is different from the published extension id which we changed in version 1.2.0.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+- Allow quotes in cache entries. [#1124](https://github.com/microsoft/vscode-cmake-tools/issues/1124) [@tmaslach](https://github.com/tmaslach)
+- Fix default preferred generators detection logic. [#1084](https://github.com/microsoft/vscode-cmake-tools/issues/1084)
+- Fix host and target platform information for VS kits. [#964](https://github.com/microsoft/vscode-cmake-tools/issues/964)
+- Fix error caused by duplicate project structure. [#587](https://github.com/microsoft/vscode-cmake-tools/issues/587) [@SchweizS](https://github.com/SchweizS)
+- Disable launchTarget key binding while debugging. [#1170](https://github.com/microsoft/vscode-cmake-tools/issues/1170)
+- Skip configuring when cache is present and according setting is on. [#984](https://github.com/microsoft/vscode-cmake-tools/issues/984)
+- Remove deprecated cmake.useCMakeServer setting. [#1059](https://github.com/microsoft/vscode-cmake-tools/issues/1059)
+- Trigger automatic CMake configure on CMakeLists.txt save. [#1187](https://github.com/microsoft/vscode-cmake-tools/issues/1187) [@Yuri6037](https://github.com/Yuri6037)
+- Silently scannig for kits:
```
Comment (bobbrow) [inline CHANGELOG.md:33]: [sp] "scanning"

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,43 @@
...
+    - when there is no available kits. [PR #1192](https://github.com/microsoft/vscode-cmake-tools/pull/1192)
```
Comment (bobbrow) [inline CHANGELOG.md:34]: [sp] is -> are

Comment (andreeis) [reply CHANGELOG.md:34]: I meant there is no available kits json file (singular). I updated.

**Pull request merged**